docker/Dockerfile-dev

```diff
@@ -1,5 +1,8 @@
 FROM python:3.5-alpine
 
+# TODO: Does this need to be done on Dockerfile-prod?
```

Comment:
Yup, these were added because of `requirements.txt` it seems and the Dockerfile-prod also installs requirements txt.

There is a command ([apk del](https://wiki.alpinelinux.org/wiki/Alpine_Linux_package_management#Remove_a_Package)) to remove these packages so that after `pip install -r requirements.txt` runs you can uninstall them to reduce the amount of stuff in the production container.